lib/config.js

```diff
@@ -49,8 +48,9 @@ module.exports = function (options) {
       log.warn('config', 'Authentication details missing from database URL')
     }
   } else {
-    log.error('config', 'options.dbUrl must be set. PouchDB not supported at this point.')
-    // log.warn('config', 'No database URL provided, falling back to PouchDB')
+    log.info('config', 'No database URL provided, falling back to PouchDB')
+    config.db.prefix = '.hoodie/data/'
```

Comment:
I think it's better to use the absolute path here: `config.db.prefix = path.join(projectPath, '.hoodie/data/')`